src/com/javarush/task/task04/task0404/Cat.java

```diff
@@ -0,0 +1,27 @@
+package com.javarush.task.task04.task0404;
+
+/*
+Учет котов
+*/
+
+public class Cat {
+    private static int catsCount = 0;
+
+    public static void addNewCat() {
+        //напишите тут ваш код
+        Cat.catsCount = Cat.catsCount + 1;
```

Comment:
Если обращаешься к статической переменной внутри класса к которому обращаешься, используй this.
```suggestion
        this.catsCount = this.catsCount + 1;
```
```suggestion
        this.catsCount += 1;
```
```suggestion
        this.catsCount++;
```

Reply:
++ добавляет единицу, как ты уже в курсе (ты используешь это в цикле for-i (i++))

Reply:
+= прибавляет к переменной слева значение справа